test/test_helper.rb

```diff
@@ -90,6 +90,22 @@ def assert_equal_sets(arr1, arr2)
         assert_equal Set.new(arr1), Set.new(arr2)
       end
 
+      def stub_supported_ssg(hosts, benchmark_versions = nil)
+        supported_ssgs = []
+
+        benchmark_versions ||= test_benchmark_versions
+
+        benchmark_versions.each do |bmv|
+          new_supported_ssgs = test_supported_ssgs(hosts, bmv)
+          supported_ssgs += new_supported_ssgs
+          # Stubbing by_ssg_version because some tests were failing due to
+          # the conditional assignment with the instance variable
+          SupportedSsg.stubs(:by_ssg_version).returns(bmv => new_supported_ssgs)
+        end
+        SupportedSsg.stubs(:all).returns(supported_ssgs)
+        SupportedSsg.stubs(:revision).returns('2022-04-20')
```

Comment:
nitpick: In this case it's not making a difference, but in general, I'd avoid using hardcoded value and use random date instead.

Reply:
Good point. Thanks!